packages/std/src/math/uint256.rs

```diff
@@ -558,6 +537,28 @@ mod tests {
         let _ = Uint256::from(1u32) - Uint256::from(2u32);
     }
 
+    #[test]
+    #[should_panic]
+    fn uint256_shr_works() {
+        let original = Uint256::new([
+            0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8,
+            0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 2u8, 0u8, 4u8, 2u8,
+        ]);
+
+        let shifted = Uint256::new([
+            128u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8,
+            0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 128u8, 1u8, 0u8,
+        ]);
+
+        assert_eq!(original >> 2u32, shifted);
+    }
```

Comment:
I don't understand this test. Why should this panic?

Reply:
It shouldn't. My mistake.